db/migration/src/m20240226_000003_create_account.rs

```diff
@@ -0,0 +1,124 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_table(
+                Table::create()
+                    .table(Account::Table)
+                    .if_not_exists()
+                    .col(
+                        ColumnDef::new(Account::Id)
+                            .integer()
+                            .not_null()
+                            .auto_increment()
+                            .primary_key(),
+                    )
+                    .col(ColumnDef::new(Account::ChainKey).string_len(40).not_null())
+                    .col(ColumnDef::new(Account::PacketKey).string_len(64).not_null())
+                    .to_owned(),
+            )
+            .await?;
+
+        manager
```

Comment:
There could also be a composite index which is set to unique to prevent insertion of 1-N mappings.

Reply:
But we want to allow 1-N, because an Account can have multiple announcements, right ?